tests/catch.hpp

```diff
@@ -8071,7 +8071,7 @@ namespace Generators {
 
     template<>
     auto all<int>() -> Generator<int> {
-        return range( std::numeric_limits<int>::min(), std::numeric_limits<int>::max() );
+        return range( (std::numeric_limits<int>::min)(), (std::numeric_limits<int>::max)() );
```

Comment:
Hmm. It may be worthwhile to report the issue you got which lead to this upstream.

Reply:
The fixed it upstream, I took the version from ensmallen and hence that was old and having issue, but it is resolved in the version which I changed it